components/setup/DockerInstructions.tsx

```diff
@@ -86,7 +86,8 @@ Coming Soon!
 				} else {
 					setSelfHostingCode(`
 docker pull asia.gcr.io/vibi-prod/dpu/dpu &&\n
-docker run -e INSTALL_ID=${response.data.installId} asia.gcr.io/vibi-prod/dpu/dpu
+mkdir -p ~/.config/vibinex &&\n
+docker run -e INSTALL_ID=${response.data.installId} -v ~/.config/vibinex:/app/config asia.gcr.io/vibi-prod/dpu/dpu
```

Comment:
so I somehow understood what you are trying to do here. But I still didn't get it completely. Can you please describe it?

Reply:
Mounting an external volume/directory, so that it can be used inside docker. Docker will be able to write github app auth token to a directory outside docker. This will be used when restarting/redeploying dpu so user does not have to give permission and choose repos again